README.md

```diff
@@ -106,14 +106,17 @@ Payout will happen at the end of the season according to this [spreadsheet](http
 
 ##### Kurt Rule (Illegal Lineup) #####
 - Only applies to **OUT** players
```

Comment:
Can we add that this applies to injury and bye week players?

Reply:
added byes